### ISSUE TITLE
Multiple triggers on the same line

### ISSUE THREAD

**Issue by Mikaela** [DarthGandalf/znclinker#11]
On #znc it was wondered if ZNC supports STARTTLS and I said that there are issues about it:

```
2015-06-17 16:17:19+0300 < Mikaela> #604 and #801
2015-06-17 16:17:20+0300 < ZNC-Linker> https://github.com/znc/znc/issues/604 “Support STARTTLS between ZNC and server. [$10]” (open)
2015-06-17 16:17:27+0300 < Mikaela> and #801
2015-06-17 16:17:28+0300 < ZNC-Linker> https://github.com/znc/znc/issues/801 “Support STARTTLS between clients and ZNC” (open)
```

I think that ZNC-linker could have said both on the same line separated by `|` or directly put the issues on different lines.

```
2015-10-08 19:56:30+0300 <@Mikaela> gdm85: for multiple clients, I would recommend taking a look at [[clientbuffer]], [[chanfilter]] and [[playback]]
2015-10-08 19:56:31+0300 < ZNC-Linker> http://wiki.znc.in/clientbuffer
2015-10-08 19:56:31+0300 < ZNC-Linker> http://wiki.znc.in/chanfilter
2015-10-08 19:56:31+0300 < ZNC-Linker> http://wiki.znc.in/playback
```

Could also be

```
2015-10-08 19:56:30+0300 <@Mikaela> gdm85: for multiple clients, I would recommend taking a look at [[clientbuffer]], [[chanfilter]] and [[playback]]
2015-10-08 19:56:31+0300 < ZNC-Linker> http://wiki.znc.in/clientbuffer http://wiki.znc.in/chanfilter http://wiki.znc.in/playback
```

or separated with `|` like suggested before.

_Multiple, more than one or two too._

**Comment by Mikaela** [comment]
Issue updated to be more  general and also include [[]].

**Comment by Zarthus** [comment]
Security concerns for github issues:

An user may link many issues in one line repeatedly to cause a large flood to github their servers and a load on znc-linker itself by opening many sockets.

The other is implemented in #17
